certs.js

```diff
@@ -114,19 +115,19 @@ async function generate(appDataPath = CERT_PATH) {
   // make binary executable
   fs.chmodSync(exePath, "0755")
   // execute the binary
-  await mkcert(appDataPath, exe)
+  await mkcert(appDataPath, exe, customDomain)
   console.log("Certificates generated, installed and trusted. Ready to go!")
 }
 
-async function getCerts() {
+async function getCerts(customDomain) {
```

Comment:
```suggestion
async function getCerts(customDomain = undefined) {
// if customDomain is undefined mkcert uses localhost
```
See (*undefined) at row 100.